scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -36,28 +36,208 @@ import { Root } from "mdast";
 export async function sphinxHtmlToMarkdown(options: {
   html: string;
   url: string;
-  imageDestination?: string;
+  imageDestination: string;
   // url links to a fixed version and ending in /
   // https://github.com/Qiskit/qiskit-ibm-runtime/tree/0.9.2/
-  baseSourceUrl?: string;
-  releaseNotesTitle?: string;
+  baseSourceUrl: string;
+  releaseNotesTitle: string;
 }): Promise<SphinxToMdResult> {
-  const {
-    html,
-    url,
-    imageDestination = "/images/api/",
-    baseSourceUrl,
-    releaseNotesTitle,
-  } = options;
-  const meta: PythonObjectMeta = {};
-  const isReleaseNotes = url.endsWith("release_notes.html") ? true : false;
-
+  const { html, url, imageDestination, baseSourceUrl, releaseNotesTitle } =
+    options;
   const $ = load(html);
   const $main = $(`[role='main']`);
+
+  const isReleaseNotes = url.endsWith("release_notes.html");
   const images = loadImages($, $main, url, imageDestination, isReleaseNotes);
+  if (url.endsWith("release_notes.html")) {
+    setReleaseNotesHeading($, releaseNotesTitle);
+  }
+
+  removeHtmlExtensionsInRelativeLinks($, $main);
+  removePermalinks($main);
+  removeDownloadSourceCode($main);
+  handleTabs($, $main);
+  addLanguageClassToCodeBlocks($, $main);
+  replaceSourceLinksWithGitHub($, $main, baseSourceUrl);
+  convertRubricsToHeaders($, $main);
+  processSimpleFieldLists($, $main);
+  removeColons($main);
+  preserveMathBlockWhitespace($, $main);
```

Comment:
This used to happen later in the sequence. I moved it here because it "fits" better here with the other processing.